tests/functional/firefox/os/version/test_base.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The name of this file `test_base.py` suggests to me that this is a base test class that's extended. Perhaps rename it to something like `test_all.py` or `test_common.py`?